docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -36,21 +36,21 @@ This procedure has been updated in the Camunda 8.6 release. The procedure used i
 
 ## Introduction
 
-This operational blueprint procedure is a step-by-step guide on how to restore operations in the case of a total region failure. It explains how to temporarily restore functionality in the surviving region, and how to ultimately do a full recovery to restore the dual-region setup. The operational procedure builds on top of the [dual-region AWS setup guide](/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md), but is generally applicable for any dual-region setup.
+This operational blueprint procedure is a step-by-step guide on how to restore operations in the case of a total region failure. It explains how to temporarily restore functionality in the surviving region and how to ultimately do a full recovery to restore the dual-region setup. The operational procedure builds on top of the [dual-region AWS setup guidance](/self-managed/setup/deploy/amazon/amazon-eks/dual-region.md), but is generally applicable for any dual-region setup.
 
 Before proceeding with the operational procedure, thoroughly review and understand the contents of the [dual-region concept page](./../../concepts/multi-region/dual-region.md). This page outlines various limitations and requirements pertinent to the procedure, which are crucial for successful execution.
 
 ## Disclaimer
 
-:::danger
+:::caution
 
-Running dual-region setups requires the users to be able to detect any regional failures and to implement the necessary operational procedure for failover and failback, matching their environments. The example blueprint procedure is described below.
+Running a dual-region configuration requires the users to detect and manage any regional failures and implement the operational procedure for failover and failback, matching their environments, utilizing the operational procedure below.
```

Comment:
```suggestion
Running a dual-region configuration requires users to detect and manage any regional failures, and implement the operational procedure for failover and failback that matches their environment.
```

I'm not sure we need to double-emphasize the operational procedure here; if anything, we could link to the procedure header?